std_msgs/Header header
string fusion_result
